.github/workflows/win.yml

```diff
@@ -39,26 +39,9 @@ jobs:
             libs/yaml.lib
             libs/xml2.lib
           key: win-libs-${{ hashFiles('.github/workflows/win.yml') }}-msvc-${{ env.VSCMD_VER }}
```

Comment:
This hash will no longer be sufficient, need to also include the new files that are now involved